crowdin_api/api_resources/abstract/resources.py

```diff
@@ -5,12 +5,16 @@
 
 
 class BaseResource(metaclass=ABCMeta):
-    def __init__(self, requester: APIRequester, page_size=25):
+    def __init__(self, requester: APIRequester, project_id: int, page_size=25):
         self.requester = requester
+        self.project_id = project_id
         self.page_size = page_size
         self._flag_fetch_all = None
         self._max_limit = None
 
+        if self.project_id is None:
+            raise ValueError("You must set project id for client")
```

Comment:
The `project_id` property should not be required, as there are API methods that do not accept it.